config/initializers/assets.rb

```diff
@@ -29,8 +29,6 @@
   break_string.js
   ember_jquery.js
   pretty-text-bundle.js
-  wizard-application.js
-  wizard-vendor.js
```

Comment:
We need `wizard.js` here, right?

(I think it's worth testing this branch in a production environment before merging - this kind of asset stuff is super hairy right now because we're halfway between legacy and ember-cli)